### ISSUE TITLE
Branch renders with elbow

### ISSUE THREAD

**Issue by sfsam** [git-up/GitUp#299]
After pushing master, my dev branch renders with an elbow:

Before:
![screen shot 2017-04-04 at 11 43 47 am](https://cloud.githubusercontent.com/assets/118104/24673143/12a7c5a2-192c-11e7-95d9-2c4a06e08508.png)

After: 
![screen shot 2017-04-04 at 11 38 48 am](https://cloud.githubusercontent.com/assets/118104/24672967/6d3c79be-192b-11e7-86a1-2d6c2d01b2b4.png)

**Comment by swisspol** [comment]
Not really a bug: it's always been designed to render like this even if puzzling.

**Comment by sfsam** [comment]
Ok, thanks for the response and thank you for this awesome app.

**Comment by daflair** [comment]
@swisspol Just to clarify, is that to signify that there was a shift inwards?

**Comment by swisspol** [comment]
No, no 😄 It's just how the rendering logic works. It had to handle various trade-offs IIRC and that was one of them.

**Comment by daflair** [comment]
OK, thanks!
